test/sequential/test-http-pipeline-flood.js

```diff
@@ -72,20 +88,13 @@ function child() {
   req = new Array(10241).join(req);
 
   conn.on('connect', function() {
+    //kill child after flooding
+    setTimeout(function() { conn.destroy(); }, common.platformTimeout(1000));
```

Comment:
OK, so I got it working with `child.kill()` but here's the downside: If the test fails, it basically fails by timing out (because a test failure means the flooding prevention mechanism never kicks in, which means `child.kill()` is never called). So, on CI that will be after 60 seconds or whatever. And on the command line, it will be never (or at least something annoyingly long). With the current setup, the timeout kicks in and a test failure only takes about 1 or maybe 2 seconds to come back and let you know it failed.

So I could go with `child.kill()` if we want to shave a few hundred milliseconds or whatever off the test, but I'd be inclined to leave the `setTimeout()` in there as well so that the test still runs quickly if it fails.